integration_test/agent_metrics/metadata.yaml

```diff
@@ -0,0 +1,237 @@
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+  labels:
+    state: .*
+- type: agent.googleapis.com/agent/log_entry_count
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+#  TODO(b/205585572#comment2) Add response_code when it becomes available
+#  labels:
+#    response_code: ^[0-9]+$
+#  TODO(b/170138116): Enable this metric once it is being collected.
+#- type: agent.googleapis.com/agent/log_entry_retry_count
+#  value_type: INT64
+#  kind: CUMULATIVE
+#  monitored_resource: gce_instance
+#  labels:
+#    response_code: .*
+#  optional: true
+- type: agent.googleapis.com/agent/memory_usage
+  value_type: DOUBLE
+  kind: GAUGE
+  monitored_resource: gce_instance
+- type: agent.googleapis.com/agent/monitoring/point_count
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+  labels:
+    status: .*
+- type: agent.googleapis.com/agent/request_count
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+#  TODO(b/205585572#comment2) Add response_code when it becomes available
+#  labels:
+#    response_code: ^[0-9]+$
+- type: agent.googleapis.com/agent/uptime
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+  labels:
+    version: .*
+  representative: true
+- type: agent.googleapis.com/cpu/load_15m
+  value_type: DOUBLE
+  kind: GAUGE
+  monitored_resource: gce_instance
+- type: agent.googleapis.com/cpu/load_1m
+  value_type: DOUBLE
+  kind: GAUGE
+  monitored_resource: gce_instance
+- type: agent.googleapis.com/cpu/load_5m
+  value_type: DOUBLE
+  kind: GAUGE
+  monitored_resource: gce_instance
+- type: agent.googleapis.com/cpu/utilization
+  value_type: DOUBLE
+  kind: GAUGE
+  monitored_resource: gce_instance
+  labels:
+    cpu_number: .*
+    cpu_state: .*
+- type: agent.googleapis.com/disk/bytes_used
+  value_type: DOUBLE
+  kind: GAUGE
+  monitored_resource: gce_instance
+  labels:
+    device: .*
+    state: .*
+- type: agent.googleapis.com/disk/io_time
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+  labels:
+    device: .*
+- type: agent.googleapis.com/disk/merged_operations
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+  labels:
+    device: .*
+    direction: .*
+  platform: linux
+- type: agent.googleapis.com/disk/operation_count
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+  labels:
+    device: .*
+    direction: .*
+- type: agent.googleapis.com/disk/operation_time
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+  labels:
+    device: .*
+    direction: .*
+- type: agent.googleapis.com/disk/percent_used
+  value_type: DOUBLE
+  kind: GAUGE
+  monitored_resource: gce_instance
+  labels:
+    device: .*
+    state: .*
+- type: agent.googleapis.com/disk/read_bytes_count
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+  labels:
+    device: .*
```

Comment:
It's missing `disk/weighted_io_time`

Reply:
Changed in: [f2495a7](https://github.com/GoogleCloudPlatform/ops-agent/pull/719/commits/f2495a72d5f14cee7af95f09f52ec3b35a036d17)